packages/npm-dependencies-validation/src/utils/packageJsonUtil.ts

```diff
@@ -41,10 +43,7 @@ async function pathContainsAnyOfFiles(
   absPath: string
 ): Promise<boolean> {
   for (const fileName of fileNames) {
-    const packageJsonDirPath = join(dirname(absPath));
-    if (await isPathExist(join(packageJsonDirPath, fileName))) {
-      return true;
-    }
+    if (await isPathExist(join(dirname(absPath), fileName))) return true;
```

Comment:
document this is written with for-of to allow early exit for async code

Reply:
Done